src/components/Sidebar/Tabs/NearByTab.tsx

```diff
@@ -0,0 +1,121 @@
+/** @jsx jsx */
+import { jsx, css } from '@emotion/react';
+import { data2schema, schema2asp, cql2asp } from 'draco-core';
+import Draco from 'draco-vis';
+
+import { GraphSpec, useModelState } from '../../../hooks/bifrost-model';
+import produce from 'immer';
+import { useState, useEffect } from 'react';
+import { VegaLite } from 'react-vega';
+import theme from '../../../theme';
+import { CircularProgress } from '@material-ui/core';
+
+const nearByTabCss = css`
+  width: 100%;
+  overflow: scroll;
+  height: 100%;
+
+  button:hover {
+    border: 10px solid ${theme.color.primary.light};
+  }
```

Comment:
If we add a border only on hover, every chart element is shifted. If you want to use this effect, we should have a `border: 10px solid transparent;` on `button`

Reply:
Here's the issue:
https://user-images.githubusercontent.com/33008436/129427849-a7baa51e-9a8b-4113-aec2-83c7c45eaa69.mov